built_json/lib/built_json.dart

```diff
@@ -37,34 +37,36 @@ abstract class Serializers {
   ///
   /// A [Serializer] must have been provided for every the object uses.
   ///
-  /// Types that are known statically can be provided via [genericType]. This
+  /// Types that are known statically can be provided via [specifiedType]. This
   /// will reduce the amount of data needed on the wire. The exact same
-  /// [genericType] will be needed to deserialize.
+  /// [specifiedType] will be needed to deserialize.
   ///
   /// Create one using [SerializersBuilder].
   ///
   /// TODO(davidmorgan): document the wire format.
-  Object serialize(Object object,
-      {GenericType genericType: const GenericType()});
+  Object serialize(Object object, {FullType specifiedType: const FullType()});
```

Comment:
> Does const FullType() mean unspecified? Define a constant for that?

const FullType() was Object which was conflated with unspecified. Actually they're not quite the same, you could conceivably add a serializer for Object, and then it's not unspecified.

Added const FullType.unspecified, distinct from Object. Updated the generation to specify FullType.unspecified instead of Object, because we don't actually use Object to mean "go find the Object serializer".